src/components/course/course-header/data/hooks/tests/useCourseRunCardData.test.jsx

```diff
@@ -10,6 +10,24 @@ import {
   MOCK_LEARNER_CREDIT_SUBSIDY,
 } from './constants';
 
+jest.mock('react', () => ({
+  ...jest.requireActual('react'),
+  useContext: jest.fn(() => ({
```

Comment:
I think this may be the first time we've mocked `useContext` in a test. Usually, we ensure the hook is rendered in a wrapper that renders the context provider with the intended context value. If you have multiple calls to `useContext` with different contexts, the proposed mocking strategy may break down a bit.

For example, see https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/1c9eb210f86c95695a480b4e7639abfb410a4ac1/src/components/course/enrollment/tests/hooks.test.jsx#L81